database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -13,12 +13,21 @@ public function up(): void
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('first_name');
+            $table->string('last_name');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->boolean('active')->default(0);
+            $table->timestamp('activated_at')->nullable();
+            $table->boolean('verified')->default(0);
+            $table->timestamp('verified_at')->nullable();
+            $table->timestamp('gender')->nullable();
+            $table->unsignedBigInteger('zone_id');
+            $table->foreign('zone_id')->references('id')->on('zones')->onDelete('cascade');
```

Comment:
Zone_id is foreignId(...) make sure that the zone migration is created first cause if the table is not found this will generate an error!!!

Reply:
Si tu as regarder l'ordre des fichiers tu verras que zone vient avant la table users